src/baldr/graphtile.cc

```diff
@@ -436,15 +440,15 @@ std::string GraphTile::FileSuffix(const GraphId& graphid, bool gzipped, bool is_
 
   // if it starts with a zero the pow trick doesn't work
   if (graphid.level() == 0) {
-    stream << static_cast<uint32_t>(std::pow(10, max_length)) + graphid.tileid() << ".gph"
-           << (gzipped ? ".gz" : "");
+    stream << static_cast<uint32_t>(std::pow(10, max_length)) + graphid.tileid()
+           << fname_suffix.c_str();
```

Comment:
dont need to actually call `c_str` as string has an ostream operator already, same below